tests/functional/v2/task/testReindex.js

```diff
@@ -0,0 +1,89 @@
+const assert = require('assert');
+const uuid = require('uuid');
+const { mpuBucketPrefix } = require('arsenal/lib/constants');
+
+const { Warp10Client } = require('../../../../libV2/warp10');
+const { ReindexTask } = require('../../../../libV2/tasks');
+const { BucketD, values } = require('../../../utils/mock/');
+const { protobuf } = require('../../../utils/v2Data');
+
+const { CANONICAL_ID, BUCKET_NAME } = values;
+const bucketCounts = [1, 1001];
+
+const bucketRecord = {
+    ops: {},
+    sizeD: 1024,
+    objD: 1,
+    inB: 0,
+    outB: 0,
+};
+
+const accountRecord = {
+    ops: {},
+    sizeD: 2048,
+    objD: 2,
+    inB: 0,
+    outB: 0,
+};
+
+// eslint-disable-next-line func-names
+describe('Test ReindexTask', function () {
+    this.timeout(1200000);
```

Comment:
Isn't this timeout too high for the expected run time of this functional test?

Reply:
yeah this looks like a debug value that made it in